session.go

```diff
@@ -177,15 +203,18 @@ func (s *Session) recvMsg() error {
 	}
 
 	const ncNamespace = "urn:ietf:params:xml:ns:netconf:base:1.0"
+	const notifNamespace = "urn:ietf:params:xml:ns:netconf:notification:1.0"
```

Comment:
minor nit make this a const block

```suggestion
	const (
		ncNamespace = "urn:ietf:params:xml:ns:netconf:base:1.0"
		notifNamespace = "urn:ietf:params:xml:ns:netconf:notification:1.0"
	)
```